src/biokbase/narrative/common/narrative_ref.py

```diff
@@ -1,9 +1,7 @@
 """Describes a Narrative Ref and has utilities for dealing with it."""
 
 from biokbase.narrative import clients
-from biokbase.workspace.baseclient import ServerError
-
-from .exceptions import WorkspaceError
+from biokbase.narrative.common.exceptions import ServerError, WorkspaceError
```

Comment:
get server errors from the kbase exceptions file so that if (for whatever reason) the ServerErrors got moved, the import statement in `exceptions.py` could be updated and nothing else would need to be touched.